configs/platforms/aix-6.1-ppc.rb

```diff
@@ -34,12 +22,24 @@
     plat.provision_with("rpm -Uvh --replacepkgs --nodeps #{name}")
   end
 
-  # We use --force with rpm because the pl-gettext and pl-autoconf
-  # packages conflict with a charset.alias file.
-  #
-  # Until we get those packages to not conflict (or we remove the need
-  # for pl-autoconf) we'll need to force the installation
-  #                                         Sean P. McD.
-  plat.install_build_dependencies_with "rpm -Uvh --replacepkgs --force "
+  # Bootstrap yum
+  plat.provision_with "curl -O http://pl-build-tools.delivery.puppetlabs.net/aix/yum_bootstrap/rpm.rte && installp -acXYgd . rpm.rte all"
+  plat.provision_with "curl http://pl-build-tools.delivery.puppetlabs.net/aix/yum_bootstrap/openssl-1.0.2.1500.tar | tar xvf - && cd openssl-1.0.2.1500 && installp -acXYgd . openssl.base all"
+  plat.provision_with "rpm --rebuilddb && updtvpkg"
+  plat.provision_with "mkdir -p /tmp/yum_bundle && cd /tmp/yum_bundle/ && curl -O http://pl-build-tools.delivery.puppetlabs.net/aix/yum_bootstrap/yum_bundle.tar && tar xvf yum_bundle.tar && rpm -Uvh /tmp/yum_bundle/*.rpm"
+
+  # Use artifactory mirror for AIX toolbox packages
+  plat.provision_with "/usr/bin/sed 's/enabled=1/enabled=0/g' /opt/freeware/etc/yum/yum.conf > tmp.$$ && mv tmp.$$ /opt/freeware/etc/yum/yum.conf"
+  plat.provision_with "echo '[AIX_Toolbox_mirror]\nname=AIX Toolbox local mirror\nbaseurl=https://artifactory.delivery.puppetlabs.net/artifactory/rpm__remote_aix_linux_toolbox/RPMS/ppc/\ngpgcheck=0' > /opt/freeware/etc/yum/repos.d/toolbox-generic-mirror.repo"
+  plat.provision_with "echo '[AIX_Toolbox_noarch_mirror]\nname=AIX Toolbox noarch repository\nbaseurl=https://artifactory.delivery.puppetlabs.net/artifactory/rpm__remote_aix_linux_toolbox/RPMS/noarch/\ngpgcheck=0' > /opt/freeware/etc/yum/repos.d/toolbox-noarch-mirror.repo"
+  plat.provision_with "echo '[AIX_Toolbox_61_mirror]\nname=AIX 61 specific repository\nbaseurl=https://artifactory.delivery.puppetlabs.net/artifactory/rpm__remote_aix_linux_toolbox/RPMS/ppc-6.1/\ngpgcheck=0' > /opt/freeware/etc/yum/repos.d/toolbox-61-mirror.repo"
+
+  # Install build dependencies
+  plat.provision_with "yum install -y coreutils autoconf gawk-3.1.3-1 pkg-config-0.19-6 rsync sed make tar glib2 perl zlib-1.2.3-4 zlib-devel-1.2.3-4"
+
+  # For now, use pl-cmake from pl-build-tools. Work around harcoded path in pl-cmake toolchain file.
+  #plat.provision_with "/opt/freeware/bin/sed -i 's|/opt/pl-build-tools|/opt/freeware|g' /opt/pl-build-tools/pl-build-toolchain.cmake"
```

Comment:
This all looks great, but let's get rid of this comment

Reply:
Yep, good catch!

Reply:
Updated to remove the comment